harvester_e2e_tests/integration/test_images.py

```diff
@@ -0,0 +1,212 @@
+
+from harvester_e2e_tests.apis import test_images, test_volumes
+
+from time import sleep
+
+import pytest
+import os
+
+
+pytest_plugins = [
+    "harvester_e2e_tests.fixtures.api_client",
+    'harvester_e2e_tests.fixtures.image',
+    'harvester_e2e_tests.fixtures.vm',
```

Comment:
I don't see any fixtures are requested from `vm` fixture.

Reply:
Thanks for the review. Remove uncessary `vm` fixture